### ISSUE TITLE
Default Content Country is Disregarded for Search Results

### ISSUE THREAD

**Issue by zbczyvgy** [TeamNewPipe/NewPipe#1744]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

NewPipe does not respect default content country selection for search results, only "Trends" are affected, the rest is displayed accordingly to the current geolocation. "Trends" title is also translated to the language of current geolocation.

**Comment by MaX-Lo** [comment]
Do you have an example for "Trends" being translated unintended? Tested some locations and can't reproduce this behaviour.

**Comment by zbczyvgy** [comment]
Try using app with a Russian IP. It would then become "Тренды", even though the Android locale is English.

**Comment by colemickens** [comment]
I am in Sweden and the header was wrong. I went into the settings and re-tapped English, then changed tabs and changed tabs back again, and it was rendered in English as expected.

In fact, while writing this, I discovered that I can reliably repro it:
1. Close the app completely.
2. Open the app. The feed starts to load with the spinner. While it's loading it says Trending.
3. The feed finishes loading, the title now is in Swedish (contrary to my settings in the app).
4. If I change tabs away, then back, it says "Trending" as expected.

The results look fine though.
